### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by bergarces**

### CODE REVIEW

File: packages/adapters-library/src/scripts/generateAdapter.test.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ describe('generateAdapter function snapshots', () => {
   generateAllOutcomes().forEach((outcomes, index) => {
     it(`should match snapshot for outcomes ${index}`, async () => {
       const blankTemplate = await readBlankTemplate(
-        'packages/adapters-library/src/adapters/blank/blankTemplate/blankAdapterForCli/blankAdapter.ts',
+        'src/adapters/blank/blankTemplate/blankAdapterForCli/blankAdapter.ts',
```
Comment (jpsains) [inline packages/adapters-library/src/scripts/generateAdapter.test.ts:95]: did you mean to change that?

Comment (jpsains) [reply packages/adapters-library/src/scripts/generateAdapter.test.ts:95]: Does it work?

Comment (bergarces) [reply packages/adapters-library/src/scripts/generateAdapter.test.ts:95]: Yes, otherwise the test fails. Jest paths seem to work different.

Comment (jpsains) [reply packages/adapters-library/src/scripts/generateAdapter.test.ts:95]: ok saw slack

**Review by jpsains — APPROVED** [review]

**Pull request merged**